style.css

```diff
@@ -148,34 +150,34 @@
     margin-bottom: -16px;
   }
 
-  section p {
+  .section p {
     font-style: normal;
     font-weight: 400;
     font-size: 16px;
     line-height: 24px;
   }
 
-  section .social-media {
+  .section .social-media {
     margin-top: -16px;
   }
 
-  section .social-media ul {
+  .section .social-media ul {
     display: flex;
     justify-content: space-between;
     width: 170.17px;
     height: 20px;
     list-style: none;
   }
 
-  section .social-media .social-text {
+  .section .social-media .social-text {
     color: #7f8cff;
     font-style: normal;
     font-weight: 500;
     font-size: 16px;
     line-height: 24px;
   }
 
-  section .social-media li {
+  .section .social-media li {
     margin-left: -40px;
   }
-}
```

Comment:
## This section is seems like a duplicate. Comment it and see that your page in mobile view looks good. You can use `@media` query for Desktop and Tablet devices.